packages/core/src/header/header.js

```diff
@@ -0,0 +1,22 @@
+/** @jsx jsx */
+import { jsx } from "@emotion/core";
+import React from "react";
+
+const Header = props => {
+  const { parents = [], onClick, title = "", getStyles } = props;
+  return (
+    <div css={getStyles("header", props)}>
+      {parents.length > 0 && (
+        <>
+          <span css={getStyles("headerBackIcon", props)} onClick={onClick}>
+            ⬅️
```

Comment:
I'm guessing we're adding this later, but with this solution you cannot update the icon.

Reply:
do you think it should be overridden by classnames or a renderer?

Reply:
a className won't override the child of the span, it will only override the style.
You want to change the content which has to be a component.